### ISSUE TITLE
In-Office Video Precence

### ISSUE THREAD

**Issue by CyborgMaster** [speak/issues#12]
A couple ideas for video as you work on it (most of which you've probably already considered).  I really loved the Sqwiggle interface.  I can see how the bubbles would be great, especially for someone with only one monitor because they are much more out of the way, but everyone on our team has a second monitor on which we would run Sqwiggle full-time, full-screen when we are "in the office."  This allowed us to have a very real office presence.  We could see at a glance to the left or right of our main monitor who was at their desk, or who was currently collaborating.  I would love to have some way to maintain that "in-office" presence; it was great for team cohesion.   So whatever you work out for video, I would recommend some sort of "always-on" picture presence, coupled with a nice big (we always did full screen) video conversation view when talking, like you currently have in Sqwiggle.

**Comment by anthonyrusso91** [comment]
This is true for us as well. We operate across two coasts, so it is awesome for us to have the presence of the other office via those still shots. Also, we love that we can then all do a theme for the day, ie. today is everyone wear sunglasses day on sqwigz 8-)

**Comment by ryanjcollier** [comment]
+1 Absolutely!
